test/e2e/resize_nodes.go

```diff
@@ -372,18 +410,19 @@ var _ = Describe("Nodes", func() {
 					break
 				}
 			}
-			Logf("Setting network partition on %s", node.Name)
+			By(fmt.Sprintf("cause network partition on node %s", node.Name))
+
 			dropCmd := fmt.Sprintf("sudo iptables -I OUTPUT 1 -d %s -j DROP", testContext.CloudConfig.MasterName)
```

Comment:
It drops all packets outbound from the node to the master.  Note that this will not work when we have multi-master (pretty soon).